shared/specs/notifications-openapi.yaml

```diff
@@ -12,7 +12,7 @@ servers:
 security:
   - BearerToken: []
 paths:
-  /api/notifications:
+  /notifications:
```

Comment:
not sure about that. there is no 'notification' route in the router code.

Reply:
The `/api` is fixed for all.
The api-segment, e.g. `/notifications` is resolved using the discoverApi based on the plugin id.
See https://github.com/backstage/backstage/blob/master/packages/core-plugin-api/src/apis/definitions/DiscoveryApi.ts#L34